src/main/java/uk/gov/hmcts/reform/blobrouter/config/ServiceConfiguration.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.blobrouter.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@ConfigurationProperties
+public class ServiceConfiguration {
+
+    @Value("service")
+    private Map<String, ServiceConfig> servicesConfig;
+
+    public Map<String, ServiceConfig> getServicesConfig() {
+        return servicesConfig;
+    }
+
+    public void setServicesConfig(List<ServiceConfig> servicesConfig) {
+        this.servicesConfig = servicesConfig
+            .stream()
+            .collect(Collectors.toMap(ServiceConfig::getName, config -> config));
```

Comment:
Let's make it smarter and adapt functional interfaces as per recommendation. So first bit is already using the reference, second one is screaming for `Function::identity` or whichever the signature is. Of course - giving stream collector allows

Reply:
Done